secure_fw/partitions/crypto/crypto_key_derivation.c

```diff
@@ -378,19 +425,23 @@ psa_status_t tfm_crypto_key_derivation_output_key(psa_invec in_vec[],
 
```

Comment:
`CRYPTO_IN_OUT_LEN_VALIDATE(in_len, 2, 2, out_len, 1, 1);` must be changed to
`CRYPTO_IN_OUT_LEN_VALIDATE(in_len, 2, 2, out_len, 2, 2);`

Reply:
Fixed